### ISSUE TITLE
different symbionts

### ISSUE THREAD

**Issue by juanelgamer456** [nonamecrackers2/crackers-wither-storm-mod#1302]
since i've been told the wither symbiont is not only created intentionally by the ws but also is the same from every encounter during the adventure, so i got an idea

what if there was different stages of the mob? this would make sense since the wither storm tries to make its chosen one stronger for every new encounter with the player
if i do remember right, there are 3 times we fight the symbiont during the game, the current symbiont could be the stage 2 or 1, and later in higher stages he reciebes more hp, more hit points, and more special abilities than just the evoker spell, the arrows, the fireballs, and the shulker orbs

some ideas i had for new stages are:
-an ability where the symbiont tracts the players close to himself for a momment to release an explosion
-an ability to spawn tentacles close to the zone where battling to grab the player
-the symbiont would able to spawn many more sickened mobs than in previous stages
-an ability that allows the symbiont to summon and shoot a single explosive wither skull every some time
-and the current abilites would increase their quantity with the stages

i know this sounds like a mob already too difficult to fight, so an option to disable the different staged symbiont would be nice

i hope this idea wont sound dumb or annoying, thanks for cracker or anyone who takes their time to read the github

**Comment by GearsawStudios** [comment]
This could pose issues with SMPs trying to grind books, Perhaps tie it to phase and not fight amount? Also the one inside the storm is intentionally weaker since there are so many hazards.

**Comment by juanelgamer456** [comment]
i think the last symbiont fight could still be harder, but aside that

as i said, it should be configurable, wich would solve both problems
now, if an option to enable and disable these stages is something too complex to code, (wich i would totally understand)

your idea to change the symbiont depending on the phase would work well, (even if it would not make sense seeing the ws sending the symbiont with the same power level as before just because is still in the same phase)

but there is a problem, the minimun phase the ws need yo spawn symbionts is phase 6, wich only left us with phase 6, 7, and inside thw bowels
wich ends up not changing my idea too much..

**Comment by GearsawStudios** [comment]
> i think the last symbiont fight could still be harder, but aside that
> 
> as i said, it should be configurable, wich would solve both problems now, if an option to enable and disable these stages is something too complex to code, (wich i would totally understand)
> 
> your idea to change the symbiont depending on the phase would work well, (even if it would not make sense seeing the ws sending the symbiont with the same power level as before just because is still in the same phase)
> 
> but there is a problem, the minimun phase the ws need yo spawn symbionts is phase 6, wich only left us with phase 6, 7, and inside thw bowels wich ends up not changing my idea too much..

No, it starts at 5.5. 6 is split into 3 and 7 is 9 heads. Also, it doesn't matter since you fight it in 5.5, 6/7 and in the bowels (as somebody who spent half a year hardcore running this mod before winning, doesn't need a buff) which fills the 3 times you fight it. I think it'd make sense as the storm would send stronger symbiotes as it grows stronger rather than amount killed. Either way, I like the idea since it feels relatively lackluster to fight the same boss twice with no changes besides the ground being destroyed more. The third fight is good as is, shorter, more violent attacks and is easily one cycled.

**Comment by juanelgamer456** [comment]
> > i think the last symbiont fight could still be harder, but aside that
> > as i said, it should be configurable, wich would solve both problems now, if an option to enable and disable these stages is something too complex to code, (wich i would totally understand)
> > your idea to change the symbiont depending on the phase would work well, (even if it would not make sense seeing the ws sending the symbiont with the same power level as before just because is still in the same phase)
> > but there is a problem, the minimun phase the ws need yo spawn symbionts is phase 6, wich only left us with phase 6, 7, and inside thw bowels wich ends up not changing my idea too much..
> 
> No, it starts at 5.5. 6 is split into 3 and 7 is 9 heads. Also, it doesn't matter since you fight it in 5.5, 6/7 and in the bowels (as somebody who spent half a year hardcore running this mod before winning, doesn't need a buff) which fills the 3 times you fight it. I think it'd make sense as the storm would send stronger symbiotes as it grows stronger rather than amount killed. Either way, I like the idea since it feels relatively lackluster to fight the same boss twice with no changes besides the ground being destroyed more. The third fight is good as is, shorter, more violent attacks and is easily one cycled.

you right its at 5.5, my bad
i mostly search for a lil bit more difficult miniboss, but NOT because the current fight is "too easy for me" or anything

**Comment by GrandmaPork** [comment]
I think that we only need 3 separate symbionts. An early zombie variant, then after WS is blown up once we have a stronger zombie or skeleton or whatever, then, finally the bowels variant which could have a beam and stuff

**Comment by myron-obsolete-account** [comment]
> I think that we only need 3 separate symbionts. An early zombie variant, then after WS is blown up once we have a stronger zombie or skeleton or whatever, then, finally the bowels variant which could have a beam and stuff

I believe the bowels symbiont is only a small side quest because it's death animation is also just the default one, and you only need to defeat it for the next CMD block hit. the main objective still is the command block itself.

**Comment by GrandmaPork** [comment]
I mean yeah but I’m saying that the wither storm could spawn in a stronger symbiont since the previous one was defeated

**Comment by BluePineappleGuy** [comment]
ideas for varients:
zombie, the og
skeleton, could maybe have exclusive projectile attacks
creeper, more explosive attacks
spider, because i'm feeling mean and there aren't a whole lot of spider boss mods anyway

**Comment by juanelgamer456** [comment]
> ideas for varients: zombie, the og skeleton, could maybe have exclusive projectile attacks creeper, more explosive attacks spider, because i'm feeling mean and there aren't a whole lot of spider boss mods anyway

this could work better too, the ws sees that a zombie is not enough so it sends different syimbiont-ed mobs

**Comment by juanelgamer456** [comment]
> I think that we only need 3 separate symbionts. An early zombie variant, then after WS is blown up once we have a stronger zombie or skeleton or whatever, then, finally the bowels variant which could have a beam and stuff

but this sounds really good too

**Comment by juanelgamer456** [comment]
i think these ideas are more than just about changing the AI

**Comment by GrandmaPork** [comment]
The one within the bowels is already different I believe so changing the model a bit could be cool.
